tests/slurm-tests/slurm_test_llama_nemotron_super_49B_v1._5_evals/recipe.py

```diff
@@ -0,0 +1,343 @@
+# Copyright (c) 2025, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import argparse
+
+# Download model and ruler on cluster
+import subprocess
+
+from nemo_skills.dataset.prepare import prepare_datasets
+from nemo_skills.pipeline.cli import convert, eval, generate, run_cmd, sft_nemo_rl, train, wrap_arguments
+
+
+def setup(workspace, cluster, expname_prefix):
+    # download models
+    cmd = (
+        f"huggingface-cli download nvidia/Llama-3_3-Nemotron-Super-49B-v1_5 --local-dir {workspace}/Llama-3_3-Nemotron-Super-49B-v1_5 && "
+        f"huggingface-cli download Qwen/Qwen2.5-32B-Instruct --local-dir {workspace}/Qwen2.5-32B-Instruct"
+    )
+    run_cmd(
+        ctx=wrap_arguments(cmd),
+        cluster=cluster,
+        expname=f"{expname_prefix}-download-models",
+        log_dir=f"{workspace}/download-assets",
+    )
+
+    # Update config to support 128k
+    cmd = (
+        f'jq \'. + {{"rope_scaling": {{"type": "yarn", "factor": 4.0, "original_max_position_embeddings": 32768}}}}\' '
+        f"{workspace}/Qwen2.5-32B-Instruct/config.json > {workspace}/Qwen2.5-32B-Instruct/config_tmp.json && "
+        f"mv {workspace}/Qwen2.5-32B-Instruct/config_tmp.json {workspace}/Qwen2.5-32B-Instruct/config.json"
+    )
+
+    run_cmd(
+        ctx=wrap_arguments(cmd),
+        cluster=cluster,
+        expname=f"{expname_prefix}-patch-qwen-config",
+        log_dir=f"{workspace}/download-assets",
+        run_after=f"{expname_prefix}-download-models",
+        container="nemo",
+    )
+
+    ruler_data_cmd = f"""
```

Comment:
let's remove this and just add a comment at the beginning of this file saying that this command should be run beforehand